tests/unit/models/transactions/test_permissioned_domain_set.py

```diff
@@ -0,0 +1,56 @@
+from unittest import TestCase
+
+from xrpl.models.exceptions import XRPLModelException
+from xrpl.models.transactions.deposit_preauth import Credential
+from xrpl.models.transactions.permissioned_domain_set import PermissionedDomainSet
+from xrpl.utils import str_to_hex
+
+_ACCOUNT_1 = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_ACCOUNT_SUBJECT = "rNdY9XDnQ4Dr1EgefwU3CBRuAjt3sAutGg"
+_VALID_CREDENTIAL_TYPE = str_to_hex("Passport")
+_VALID_URI = str_to_hex("www.my-id.com/username")
+
+
+class TestPermissionedDomainSet(TestCase):
```

Comment:
Should add the no-dup test as well

Reply:
included in [55239ba](https://github.com/XRPLF/xrpl-py/pull/773/commits/55239ba91ff8a670a2455838290ed94e9dfd6106)